doc/issue_impact.md

```diff
@@ -0,0 +1,25 @@
+# Issue Impacts
+
+axe-core assigns an impact according to our assessment of the likely impact of an issue on a user with a disability that would be affected by this issue.
+
+In any given context the actual impact for the user could be lower and in some instances, it could be higher.
+
+For this reason, we encourage users of tools to evaluate each individual issue and assess the impact in the context of their application or content.
+
+## Definitions
+
+### Minor
+
+Considered to be a nuisance or an annoyance bug. Prioritize fixing if the fix only takes a few minutes and the developer is working on the same screen/feature at the same time, otherwise the issue should not be prioritized. Will still get in the way of compliance if not fixed. Should be very infrequent.
```

Comment:
> Should be very infrequent

What exactly should be infrequent? An issue being reported with minor impact or fixing the issue?

Reply:
was a comment on the frequency of the impact within the rules, have removed because it was not clear.